.github/vale/Vocab/Backstage/accept.txt

```diff
@@ -92,6 +92,7 @@ elasticsearch
 esbuild
 eslint
 etag
+ent
```

Comment:
Not needed anymore right? This should always be wrapped in a code block anyway so let's avoid adding it.